### PR TITLE
Add Persian translation for `Readme.md`

### PR DESCRIPTION
**Pull request by Mobinshahidi**
### What was done?
- Added Persian (Farsi) translation to `Readme.md`.
- Updated the translation table and included a link to `Farsi.md`.

### What are the next steps?
- Translate `maintainer_guide.md` to Persian.
- Translate `terminal_tutorial.md` to Persian.

### CODE REVIEW

**Comment by Mobinshahidi** [comment]
Hi, how are you?
Can you review my changes please so i can start translating rest of the readme?

@CandidDeer

**Comment by CandidDeer** [comment]
@Mobinshahidi Since I don't know the Persian language, I'm afraid I can't really give any input, sorry!

**Comment by Mobinshahidi** [comment]
So if someone who knows persian, review this PR and approved its ok?
@CandidDeer

**Review by TenTraicion — CHANGES_REQUESTED** [review]
Hello there @Mobinshahidi, Thanks a bunch for the translation, i really appreciate your work. Kindly remove the html tags or any other element except markdown syntax and text, and this file is good to be merged. Lo0king forward to terminal tutorial translation. 

also kindly rename the file name to: PERSIAN.md and also link the file in translation.md/ Thanks.

**Comment by Mobinshahidi** [comment]
Thank you so much for your review @TenTraicion.
The html tags is because that persian language is RTL and because of that i used the html tags and without them is not looking good.
Whats your opinion?

**Comment by TenTraicion** [comment]
@Mobinshahidi, hello there. I understand your situation. But since this year, we discourage contributors to use any kind of external tags/codes unless it's approved by github markdown language. So kindly use only Github Markdown for .md files.

**Comment by Mobinshahidi** [comment]
@TenTraicion hi how are you?
i did what you told me, please check them again.
thanks

File: translations/README/PERSIAN.md [hunk 1]
```diff
@@ -0,0 +1,384 @@
...
+  - [قدم پنجم: تمپلیت کارد را کپی کنید](#step-5)
+  - [قدم ششم: تغییرات خودتان را اعمال کنید](#step-6)
+  - [قدم هفتم: تغییرات خود را کامیت بزنید](#step-7)
+  - [قدم هشتم: تغییرات خود را در گیتهاب پوش کنید](#step-8)
+  - [قدم نهم: یک پول ریکوست ثبت کنید](#step-9)
+  - [قدم دهم: جشن بگیرید](#step-10)
+- [مرحله بعدی](قدم-بعدی)
+- [قدردانی ها](#acknowledgements)
+- [مجوز](#license)
+- [100 مشارکت کننده برتر](#top-100-contributors)
```
Comment (TenTraicion) [inline translations/README/PERSIAN.md:44]: need to fix the links in this content table. most of the links are not working!

File: translations/README/PERSIAN.md [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+| :---------------------------------------------: | :---------------------------------------: | :-----------------------------------------: | :---------------------------------------: | :---------------------------------------: |
+| [French (Français)](/translations/README/FRENCH.md) |  [German (Deutsch)](/translations/README/GERMAN.md)  |      [Hindi (हिंदी)](/translations/README/HINDI.md)      | [Italian (Italiano)](/translations/README/ITALIAN.md) | [Japanese (日本語)](/translations/README/JAPANESE.md) |
+  [Korean (한국어)](/translations/README/KOREAN.md)  | [Polish (Polski)](/translations/README/POLISH.md) | [Portuguese (Portuguese)](/translations/README/PORTUGUESE.md) | [Russian (Русский)](/translations/README/RUSSIAN.md) |  [Serbian (Српски)](/translations/README/SERBIAN.md)  |
+  [Spanish (Español)](/translations/README/SPANISH.md) | [Turkish (Türkçe)](/translations/README/TURKISH.md) | [Ukrainian (українська)](/translations/README/UKRAINIAN.md) | [Norwegian (Norsk)](/translations/README/NORWEGIAN.md) | [Persian (فارسی)](/translations/README/FARSI.md)
```
Comment (TenTraicion) [inline translations/README/PERSIAN.md:86]: the link in Persian language should be `/translation/README/PERSIAN.md` according to the file name. also don't put it here, as it is the Persian translation file, you need to fix the link in other files where you've mentioned this translation.

File: translations/README/PERSIAN.md [hunk 3]
```diff
@@ -0,0 +1,384 @@
...
+
+1. در اکانت گیتهاب خود لاگین کنید. اکر هنوز اکانتی ندارید [به گیتهاب جوین شوید](https://github.com/join). من پیشنهاد میکنم که آموزش [GitHub Hello World](https://guides.github.com/activities/hello-world/) را قبل از ادامه دادن، خوانده باشید.
+2. برنامه [GitHub Desktop](https://desktop.github.com/) را دانلود کنید.
+   - از سوی دیگر اگر با گیت و خط فرمان راحت‌‌ تر هستید،  [این لینک آموزش CLI است](/terminal_tutorial.md).
+   - اگر شما از [VS Code](https://code.visualstudio.com/ 'Visual Studio Code website') استفاده میکنید، میتوانید از این ادیتور برای ادامه استفاده کنید.
+   - همچنان راحت ترین راه و ساده ترین راه استفاده از برنامه [GitHub Desktop](https://desktop.github.com/) است.
+
+حالا که شما آماده به کار شدید، بیایید تا مشارکت در این پروژه را شروع کنیم.
+
+[↑ به بالا برو ↑](#)
```
Comment (TenTraicion) [inline translations/README/PERSIAN.md:106]: the go up link in your translation should be something like: `#فهرست-دسترسی-سریع`

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@Mobinshahidi, kindly fix the mentioned issues and we are good to merge this file. Thanks.

**Pull request closed without merging**